dev/TabView/TabView.cpp

```diff
@@ -233,14 +233,42 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
         // Now that ListView exists, we can start using its Items collection.
         auto items = TabItems();
         auto numItemsToCopy = static_cast<int>(items.Size());
-        if (auto lvItems = listView.Items())
+        bool itemsSourceDoesNotExists = listView.ItemsSource() == nullptr;
```

Comment:
[Don't add unnecessary "== nullptr"](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es87-dont-add-redundant--or--to-conditions):

```suggestion
        bool itemsSourceDoesNotExists = !listView.ItemsSource();
```

But also I'd avoid the local and just put this check in the if condition below.

Reply:
```
if(!listView.ItemsSource())
```

---
In reply to: [338187053](https://github.com/microsoft/microsoft-ui-xaml/pull/1482#discussion_r338187053) [](ancestors = 338187053)